src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -1063,13 +1063,31 @@ public static AutoCreateMode parse(String value, String defaultValue) {
                     "'skip' to skip / ignore TRUNCATE events (default), " +
                     "'include' to handle and include TRUNCATE events");
 
+    public static final Field OVERRIDE_TRANSACTION_ORDERING_DEPRECATION = Field.create("override.transaction.ordering.deprecation")
```

Comment:
Maybe change the name to `TEST.override.transaction.ordering.deprecation` and mention in description that this is test only config.